core/alsp_src/library/html_tokens.pro

```diff
@@ -604,6 +604,10 @@ dispatch_read_to_terminator(C, S, [], C)
 dispatch_read_to_terminator(C, S, [], C)
 	:-
 	C = 0'=, !.
+dispatch_read_to_terminator(C, S, [], C)
+        :-
+%        C = 0’&, !.
+        C = 38, !.
```

Comment:
I had some problems � it would skip this clause, even when I dropped all *.obp files. I�ll give it a fresh try.

On Dec 9, 2015, at 5:57 AM, Chuck Houpt <notifications@github.com> wrote:

> In core/alsp_src/library/html_tokens.pro:
> 
> > @@ -604,6 +604,10 @@ dispatch_read_to_terminator(C, S, [], C)
> >  dispatch_read_to_terminator(C, S, [], C)
> >  	:-
> >  	C = 0'=, !.
> > +dispatch_read_to_terminator(C, S, [], C)
> > +        :-
> > +%        C = 0�&, !.
> > +        C = 38, !.
> 
> The rest of the code uses 0'& instead of 38. Does 0'& not work here?
> 
> �
> Reply to this email directly or view it on GitHub.
>